.github/workflows/check-if-migration-is-required.yml

```diff
@@ -47,3 +47,5 @@ jobs:
         run: |
           # Below will exit with non-zero status if model changes are missing migrations
           sentry django makemigrations -n ci_test --check --dry-run --no-input || (echo '::error::Error: Migration required -- to generate a migration, run `sentry django makemigrations -n <some_name>`' && exit 1)
+          [ $(git status --short | grep "M migrations_lockfile.txt" | wc -l) == 1 ] && \
+            echo >&2 "::error::Error: You have not updated the migration_lockfile.txt" && exit 1
```

Comment:
Is there a command to update the lockfile? If so can we show the user what to run to update it?

Reply:
All migrations should be generated with `(get)sentry django makemigrations -n <somename>` or if it's a data migration, `(get)sentry django makemigrations -n <somename> --empty`